.craft.yml

```diff
@@ -21,3 +19,13 @@ targets:
       maven:io.sentry:sentry-android-okhttp:
       maven:io.sentry:sentry-kotlin-extensions:
       maven:io:sentry:sentry-android-fragment:
+  - name: maven
+    gradleCliPath: ./gradlew
+    mavenCliPath: scripts/mvnw
```

Comment:
is this accurate after moving the /mvn folder?

Reply:
Yes, `.mvn/` should be in the same directory as `.craft.yml`.